src/components/Datatable/Table/TableHeadCell/SortingIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+
+import { SSCIconNames } from '../../../../theme/icons/icons.enums';
+import { ColorTypes } from '../../../../theme/colors.enums';
+import { pxToRem } from '../../../../utils/helpers';
+import { Icon } from '../../../Icon';
+import { SortingIconProps } from './TableHeadCell.types';
+
+const StyledIcon = styled(Icon).attrs((props) => ({
+  color: ColorTypes.graphite2B,
+  margin: { left: 0.3 },
+  ...props,
+}))<{ $isActive?: boolean } & React.ComponentProps<typeof Icon>>`
+  font-size: ${pxToRem(10)};
+
+  ${({ $isActive }) =>
+    $isActive &&
+    css`
+      font-size: ${pxToRem(14)};
+    `};
+`;
+
+const SortingIcon: React.FC<SortingIconProps> = React.memo(
+  ({ isSorted, isSortedDesc }) => {
+    if (isSortedDesc)
+      return <StyledIcon name={SSCIconNames.sortDown} $isActive />;
```

Comment:
Does have `$` in prop name special meaning?

Reply:
A common question these times :D Yep that is the notation for transient props for styled-component (https://styled-components.com/docs/api#transient-props). That means those props are not passed to DOM

Reply:
Oh yes, now I remember, new transient props. Nice!